src/System.ValueTuple/src/System.ValueTuple.csproj

```diff
@@ -29,6 +29,9 @@
     <TargetingPackReference Include="System" />
     <TargetingPackReference Include="System.Core" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == ''">
+    <TargetingPackReference Include="System.Private.CoreLib" />
```

Comment:
> <TargetingPackReference Include="System.Private.CoreLib" /> [](start = 3, length = 60)

do we still need to reference System.Private.Corelib here? should we just reference System.Runtime?